R/show.R

```diff
@@ -191,10 +191,15 @@ expressionValue <- function (expr) {
     unlist(expr$column %||% expr$value)
 }
 
-deparseAndTruncate <- function (x, ...) {
+deparseAndFlatten <- function (x, max_length = 40, ...) {
```

Comment:
Should add an explicit test that this works correctly with max_length specified and not.